rapyuta_io/clients/device.py

```diff
@@ -320,6 +323,9 @@ def _serialize(self):
                       '_ros_workspace']:
             if getattr(self, field):
                 device['config_variables'][field[1:]] = getattr(self, field)
+        for key, value in self.config_variables.items():
+            device['config_variables'][key] = value
+        device['labels'] = self.labels
```

Comment:
question: Why did we have to loop for config_variables but for labels we assigned directly.